exporter/cat1_view_helper.go

```diff
@@ -15,29 +15,17 @@ import (
 	"github.com/projectcypress/cdatools/models"
 )
 
-func escape(i interface{}) string {
-	switch str := i.(type) {
-	case string:
-		return escapeString(str)
-	case int64:
-		return escapeString(strconv.FormatInt(str, 10))
-	case int:
-		return escapeString(strconv.Itoa(str))
-	case *int64:
-		if str != nil {
-			return escapeString(strconv.FormatInt(*str, 10))
-		}
-	case *int:
-		if str != nil {
-			return escapeString(strconv.Itoa(*str))
-		}
-	}
-	return ""
+// i64ptrToString converts the value of an *int64 to a string.
+func i64ptrToString(i *int64) string {
```

Comment:
TBH `Int64PtrToString` seems the most go-ish to me, but to avoid bikeshedding here you can just make the call